lib/wallaby/dsl.ex

```diff
@@ -1,5 +1,26 @@
 defmodule Wallaby.DSL do
-  @moduledoc false
+  @moduledoc """
+  Sets up the Wallaby DSL in a module.
+
+  All functions in `Wallaby.Browser` are now accessible without a module name
+  and `Wallaby.Browser`, `Wallaby.Element` and `Wallaby.Query` are all aliased.
+
+  ## Example
+
+  ```
+  defmodule MyPageObject do
+    use Wallaby.DSL
+
+    def register(session) do
+      session
+      |> visit("/registration.html")
+      |> fill_in(Query.text_field("Name"), with: "Chris")
+      |> fill_in(Query.text_field("email"), with: "c@keathly.io")
+      |> click(Query.button("Save"))
```

Comment:
I just updated this to use module attributes and match the rest of the documentation. I'm not sure why, but dialyzer throws a fit when I store these queries in module attributes in my personal project. It could be just me, though.

```
def set_email(session, email) do
  session
  |> fill_in(Query.css("[name=email]"), with: email)
end
```
works, but 
```
@email_field Query.css("[name=email]")

def set_email(session, email) do
  session
  |> fill_in(@email_field, with: email)
end
```
gives me the following warning
```
Function set_email/2 has no local return
```